spec/MergedWideCoverageData.csvschema.json

```diff
@@ -0,0 +1,16 @@
+{
+    "columns": {
+        "scenario": "String",
+        "set_name": "String",
+        "vaccine": "String",
+        "gavi_support": ["no vaccine", "no gavi", "total"],
+        "activity_type": ["none", "routine", "campaign", "campaign-reactive"],
```

Comment:
Could these enums be pulled out into schema files of their own? Since they are used in more than one place now.

Reply:
This would require some work on the CSVSchema parser. Opened a ticket for this: https://vimc.myjetbrains.com/youtrack/issue/VIMC-840